workflows/busco_pipeline.nf

```diff
@@ -105,8 +107,8 @@ include { BUSCO_PROTEIN_OUTPUT } from '../modules/busco_protein_output.nf'
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
 
-workflow{
-        csvData = Channel.fromPath(params.csvFile).splitCsv()
```

Comment:
could you please restore it? Otherwise the content of the csv is considered an array for the next process and not as a single input for each of the next 'parallel' processes

Reply:
Ah! Good to know! My understanding (based on the documentation) is that `Channel.fromPath()` was interchangeable with `file()`.

Reply:
apparently file() reads the csv and store the content in an array while splitCsv requires the file itself